Kong/docs.konghq.com
# Fix: Logging plugins log format + descriptions

### Description

The log format example and descriptions were missing some changes from 3.6, and had incorrectly tagged versions. 
Tested this out via the File Log plugin and compared the output to any log serializer [changelog entries](https://docs.konghq.com/gateway/changelog/). 

"JSON object considerations" was also a strange title that didn't describe the contents, which are just descriptions of each log item, so updated that to be clearer.

Issue reported on Slack.

### Testing instructions

Preview link: Check any logging plugin. For example, I used File Log:
https://deploy-preview-7948--kongdocs.netlify.app/hub/kong-inc/file-log/
https://deploy-preview-7948--kongdocs.netlify.app/hub/kong-inc/file-log/3.5.x/
https://deploy-preview-7948--kongdocs.netlify.app/hub/kong-inc/file-log/3.4.x/

### Checklist
